### ISSUE TITLE
Geting worse when training normal claasification.

### ISSUE THREAD

**Issue by erow** [ronghuaiyang/arcface-pytorch#35]
I want to see the generality of arcface. Because it's a loss function which doesn't depend on problem. But I found it got worse when I applied this in satimage classifacation.

**Comment by ronghuaiyang** [comment]
the purpurse of arcface is to get discriminative features, not to maximum the classification accuracy.

**Comment by shaqing** [comment]
then,if i use this in classification problem,it will won't boost the performance?

**Comment by erow** [comment]
I have tested it on wide datasets. Generally, it does boost performance but bot always.
